docs/get-started/nextjs.md

```diff
@@ -301,41 +301,6 @@ The default values on the stubbed router are as follows (see [globals](../essent
 ```ts
 // Default router
 const defaultRouter = {
-  push(...args) {
```

Comment:
The new recommended way to "override" these is with `mockImplementation`, in which case you don't really need to know about their default behavior (which I also think is different now) and the caveat about them missing from the actions panel is now also not true anymore.

Reply:
Oh, great catch! 🙏

Reply:
So how do you actually override one of the methods now?

```ts
// .storybook/preview.ts
import { Preview } from '@storybook/react';

const preview: Preview = {
  parameters: {
    nextjs: {
      router: {
        push: // TK: How?,
      },
    },
  },
};
```

Or do you have to use `beforeEach`?

```ts
// .storybook/preview.ts
import { Preview } from '@storybook/react';

const preview: Preview = {
  async beforeEach({ parameters }) {
    parameters.nextjs.router.push.mockImplentation(() => { /* ... */ });
  },
};
```

Reply:
Oh, wait. Maybe you import the mock?

```
// .storybook/preview.ts
import { Preview } from '@storybook/react';
import { getRouter } from '@storybook/nextjs/router';

const preview: Preview = {
  parameters: {
    nextjs: {
      router: {
        push: getRouter().push.mockImplentation(() => { /* ... */ }),
      },
    },
  },
};
```

Reply:
I guess the latter could work, but the recommended way would now be this:

```ts
// .storybook/preview.ts
import { Preview } from '@storybook/react';
import { getRouter } from '@storybook/nextjs/router.mock';

const preview: Preview = {
  async beforeEach() {
    getRouter().push.mockImplementation(() => { /* ... */ });
  },
};
```

Reply:
The functions are really best modified with `mockImplementation` and the parameters should rarely be used, except for the non-function properties like `pathname`, `query`, etc. which I've also left in.

Reply:
I think I found a format that demonstrates both clearly. But feel free to tweak it or make further suggestions.

Reply:
LGTM